src/services/squidrouter/route.ts

```diff
@@ -152,3 +150,25 @@ export async function getRouteTransactionRequest(
     data: routeResult.data,
   };
 }
+
+export async function testRoute(testingToken: InputTokenDetails, attemptedAmountRaw: string) {
```

Comment:
I think it would be better if this function would actually test to get the routes for the tokens the user might want to use. Namely USDC and USDC.e -> axlUSDC. If we don't check for these specific routes, it might happen that squidrouter is able to return something for the test route but not for the actual routes we want to take.

Reply:
Well the idea at least is that exactly! If you look at where `testRoute` is [called](https://github.com/pendulum-chain/vortex/pull/181/files#diff-777e0c2221f5feeef076c359496045ddd95d6d376a0fc1395b330f8a9b64a5bbR150) which is on confirm, it will perform the exact route the user is trying to offramp. The hardcoded parameters are the `to` token which is `axlUSDC` and the address, which shouldn't matter for the route. Also the amount is the one the user requires.

This is why I decided to put this check here and not when loading the app. There are too many variables (`from` and `amount`) to check all the possibilities, especially as we add more tokens. Let me know what you think.

Reply:
Oh I missed that, sorry! I think I was confused because the requirements in https://github.com/pendulum-chain/vortex/issues/120 all declare that these checks should happen on start of the app. But nevermind my comment then, it makes sense to check it like this 👍